components/authz-service/server/v2/system.go

```diff
@@ -66,6 +66,12 @@ func SystemPolicies() []*storage.Policy {
 				Resources: []string{"iam:rules"},
 				Projects:  []string{constants.AllProjectsID},
 			},
+			{
+				Effect:    storage.Allow,
```

Comment:
we shouldnt need this as a system policy, ill be around later tonight if you want to chat more. i think i miscommunicated earlier.